src/aiida/orm/nodes/data/code/installed.py

```diff
@@ -137,6 +137,13 @@ def validate_filepath_executable(self):
             with override_log_level():  # Temporarily suppress noisy logging
                 with self.computer.get_transport() as transport:
                     file_exists = transport.isfile(str(self.filepath_executable))
+                    if file_exists:
```

Comment:
```suggestion
                    if  (file_exists := transport.isfile(str(self.filepath_executable))):
```

Reply:
Tbh, I'm not a big fan of this walrus operator, so I didn't use it. I find the other syntax much more readable. So I'll ignore :p